qiskit_nature/second_q/algorithms/ground_state_solvers/ground_state_eigensolver.py

```diff
@@ -34,7 +34,7 @@ class GroundStateEigensolver(GroundStateSolver):
 
     def __init__(
         self,
-        qubit_converter: QubitConverter,
+        qubit_converter: QubitConverter | QubitMapper,
```

Comment:
The docstring below also needs a minor update.
The same goes for some of the other modified classes.